habitat_baselines/run.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import random
+from typing import List
+
+import numpy as np
+
+from habitat import Config, get_config
+from habitat_baselines.common.utils import get_trainer
+from habitat_baselines.config.default import get_config as baseline_cfg
+
+
+def get_exp_config(cfg_path: str, opts: List[str] = None) -> Config:
+    r"""
+    Create config object from path for a specific experiment run.
+    Args:
+        cfg_path: yaml config file path.
+        opts: list additional options or options to be overwritten.
+
+    Returns:
+        config object created.
+    """
+
+    config = Config(new_allowed=True)
+    config.merge_from_other_cfg(baseline_cfg(cfg_path))
+    task_config = get_config(config.BASE_TASK_CONFIG_PATH)
+    config.TASK_CONFIG = task_config
```

Comment:
You kill `config.TASK_CONFIG` node that was defined before. I think we discussed opposite:
`config.TASK_CONFIG.merge(task_config)`.

Reply:
Maybe `config.TASK_CONFIG` should take more priority in merging than base `task config`? Like:
```
config.TASK_CONFIG = task_config.merge(config.TASK_CONFIG)
```